Formula/x11vnc.rb

```diff
@@ -50,7 +50,6 @@ def install
 --- a/x11vnc/solid.c
 +++ b/x11vnc/solid.c
 @@ -177,7 +177,7 @@ unsigned long get_pixel(char *color) {
- 
```

Comment:
Heh, I think you've completely removed the whitespace here now. Looking at the original patch there's a single space on this line.

Sorry to be a pain about this 😢. Too much experience with minor whitespace changes in patches causing user breakage.